mythril/analysis/modules/dos.py

```diff
@@ -70,7 +70,7 @@ def _analyze_states(self, state: GlobalState) -> List[Issue]:
 
                 try:
                     self._jumpdest_count[transaction][target] += 1
-                    if self._jumpdest_count[transaction][target] == 4:
+                    if self._jumpdest_count[transaction][target] == 3:
```

Comment:
Using JUMPDEST_LIMIT - 1 might be good here, what do you think?

Reply:
Yeah, great idea!